terraform/ec2/linux/main.tf

```diff
@@ -26,6 +26,45 @@ locals {
   private_key_content = var.ssh_key_name != "" ? var.ssh_key_value : tls_private_key.ssh_key[0].private_key_pem
 }
 
+#####################################################################
+# Create EFS
+#####################################################################
+resource "aws_efs_file_system" "efs" {
+  creation_token = "predictable-${random_id.testing_id.hex}"
+  tags           = {
+    Name = "predictable-${random_id.testing_id.hex}"
```

Comment:
nit - this is not a great name. What is predictable? We're tagging the resource by naming it here.